pupil_src/shared_modules/player_methods.py

```diff
@@ -48,7 +48,7 @@ def __init__(self, data=(), data_ts=()):
                 )
             )
         elif not len(data):
-            self.data = []
+            self.data = np.asarray([])
             self.data_ts = np.asarray([])
```

Comment:
Is there a specific reason for why we use `asarray` here?
Looks like we could also use `np.array([])` or even `np.empty(0)`

Reply:
It is mostly used to mirror the other `asarray` calls below. I guess using `np.empty` would be easier to read. What do you think? I am open to changing these two lines.

Reply:
I think I have seen `np.array([])` in multiple places throughout the codebase. Maybe that's a bit more readable than `np.empty(0)`. Too bad there's no `np.empty()` :smile: